pdns/dnsdist-lua2.cc

```diff
@@ -769,6 +769,27 @@ void moreLua(bool client)
 #endif
       });
 
+    g_lua.registerFunction<void(DNSDistProtoBufMessage::*)(std::string)>("setTag", [](DNSDistProtoBufMessage& message, const std::string& strValue) {
+      message.addTag(strValue);
+     });
+
+    g_lua.registerFunction<void(DNSDistProtoBufMessage::*)(vector<pair<int, string>>)>("setTagArray", [](DNSDistProtoBufMessage& message, const vector<pair<int, string>>&tags) {
+      for (const auto& tag : tags) {
+        message.addTag(tag.second);
+      }
+     });
+
+    g_lua.registerFunction<void(DNSDistProtoBufMessage::*)(boost::optional <time_t> sec, boost::optional <uint> uSec)>("setProtobufResponseType",
+                                        [](DNSDistProtoBufMessage& message, boost::optional <time_t> sec, boost::optional <uint> uSec) {
```

Comment:
`uSec` should be a `uint32_t` here to be consistent with `DNSProtoBufMessage::setQueryTime()`. Be careful that you need to change it twice due to the `LuaWrapper` syntax.